code-challenges.test.js

```diff
@@ -22,10 +22,107 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+//Must do a red-green-refactor process:
+//First step make the test fail, expected to fail due to undefined function since function has not been defined.
+//Second step create the function, make the test pass.
+//Function will have a conditional to verify value is divisible by three.
+//The conditional statement will be number % 3 === 0, therefore divisible by three, else not divisible by three.
+//Third step refactor to optimize and make the code cleaner, make the test pass.
+
+/* =====Misspelled toEqual therefore test was not passing after defining a function======
+//First step 1:
+describe("divisibleByThree", () => {
+
+    it("function takes an input value as an argument and outputs a string if it is or not divisible by three", () => {
+
+        expect(divisibleByThree(num1)).toEaqual("15 is divisible by three")
+    })
+});
+//Test result failed (ReferenceError: divisibleByThree is not defined).
+
+//Implimenting a test for the other two cases, num2 & num3.
+describe("divisibleByThree", () => {
+
+    it("function takes an input value as an argument and outputs a string if it is or not divisible by three", () => {
+
+        expect(divisibleByThree(num2)).toEaqual("0 is divisible by three")
+    })
+});
+
+describe("divisibleByThree", () => {
+
+    it("function takes an input value as an argument and outputs a string if it is or not divisible by three", () => {
+
+        expect(divisibleByThree(num3)).toEaqual("-7 is not divisible by three")
+    })
+});
+*/
+
+//Second Attempt after correctly spelling toEqual
+//First step 1:
+describe("divisibleByThree Function", () => {
+
+    it("function takes an input value as an argument and outputs a string if it is or not divisible by three", () => {
+
+        expect(divisibleByThree(num1)).toEqual("15 is divisible by three")
+    })
+});
+//Test result failed (ReferenceError: divisibleByThree is not defined).
+
+//Implimenting a test for the other two cases, num2 & num3.
+describe("divisibleByThree Function", () => {
+
+    it("function takes an input value as an argument and outputs a string if it is or not divisible by three", () => {
+
+        expect(divisibleByThree(num2)).toEqual("0 is divisible by three")
+    })
+});
+
+describe("divisibleByThree Function", () => {
+
+    it("function takes an input value as an argument and outputs a string if it is or not divisible by three", () => {
+
+        expect(divisibleByThree(num3)).toEqual("-7 is not divisible by three")
+    })
+});
```

Comment:
The syntax looks good, but you can wrap all three expect statements in the same `it` block.